### ISSUE TITLE
Why is duffyou no longer in your repo?

### ISSUE THREAD

**Issue by pitsi** [mrgsi/repository.gsource#3]
I am a simple youtube (= the addon) user who got very annoyed after its api's bs the last couple of days.
I came across your addon from some post in reddit, I added your repo and I found out that duff you is no longer available in it. In fact, it was removed exactly 5 months ago. May I ask why?

**Comment by mrgsi** [comment]
Some users were reporting issues with different OS's so I thought it best to remove it as I am not the developer and can't provide support for it. You can still get it directly from the developer: https://palantirsource.github.io/palantir/

**Comment by pitsi** [comment]
Thank you for the clarification. Oddly, I have never seen that repo in any howto or comment!

**Comment by mrgsi** [comment]
Could be because it's a Spanish language developer.

**Comment by pitsi** [comment]
Maybe. But most howtos are in english, so what difference would it make to promote someone's addon if its dev is Spanish.
Anyway...

**Comment by pitsi** [comment]
Sorry for bringing this back up, but I have an honest question.

The addon is no longer hosted in palantir's repo and I have found it on 2 completely unknown github repos. The first has all sorts of weird addons and the second one is from a build. 
How about hosting it again in your repo? At least yours has a good reputation! 

I can provide the latest zip if needed.

**Comment by mrgsi** [comment]
I did like the add-on so I get where you are coming from, but as I mentioned it was flawed since it wasn't a multi-OS add-on. Plus, any add-ons using youtube for trailers pulls from the youtube add-on so most users would want the actual youtube add-on. Not to mention, the youtube add-on now works without api keys. I also don't like offering an add-on that has no support. If the actual developer doesn't want anything to do with it, then I don't want to touch it either. It's basically a "zombie" now. lol

**Comment by pitsi** [comment]
Your reply makes me ask too many questions! Do you mind answering them? Thank you :)

What do you mean with "it wasn't a multi-os addon"? What os(es) doesn't it support?
When did the youtube addon start working without api keys? All I know is that it can play trailers and nothing more without api keys, but I can not test it. After all, youtube (the addon) is not only for trailers or as a companion addon to the ones that play movies/series, it is a fully functional addon for youtube itself (the site).

**Comment by mrgsi** [comment]
LOL. The road to wisdom is paved with questions, grasshopper. I forget which OS was problematic, it's been too long and I've looked at way too many add-ons. I think the youtube add-on was updated with developer keys in 7.0.2.0, but I am using version 7.0.3+beta.1 without keys and it's playing videos from the search and from the popular menu option. I couldn't tell you if it works 24/7 but it has worked when I played with it. I've always used my own keys so didn't really notice until a week ago.
